other/ethhub-cftc-response.md

```diff
@@ -162,11 +162,23 @@ Ethereum has operated uninterrupted for three years as a decentralized blockchai
+
+Given the nascent markets for cryptocurrencies and longstanding laws for KYC/AML, these exchanges play a vital role in allowing investors to legally exchange Ether with USD. Even so, until their performance is tested in more market cycles, an unexpected shutdown to their operations via technical problems or business insolvency remains a significant risk to the purchase and holding of Ether.
+
+Granted, one significant caveat to the risks presented above is that investors can easily exchange Ether into other cryptocurrencies via an array of traditional exchanges, decentralized exchanges, and other services. Many of these sites, such as Shapeshift, do not require a login to exchange currencies instantaneously, making such an exchange trivial to most users. Thus, when considering the liquidity of Ether, it’s important to account for the ability to exchange any cryptocurrency (particularly other liquid ones such as Bitcoin) to USD as a secondary source of liquidity. 
+
+It’s also important to consider the risks that some of these providers represent to the ecosystem. Exchanges which don’t have regulated fiat on-ramps often are unaudited, lack proper licenses, and conduct questionable practices such as volume spoofing (An analysis of fake volume: https://medium.com/@sylvainartplayribes/chasing-fake-volume-a-crypto-plague-ea1a3c1e0b5e). These exchanges deserve further scrutiny to protect investors and institutions alike from potential misconduct. 
+
+The emergence of stable coins offers an array of opportunities and risks as well. A stable coin is a cryptocurrency that has a value pegged to a real-world asset such as USD. These coins facilitate the exchange of USD and Ether by ensuring transactions are easier to track, atomic (Ether only is received if USD is sent), and cheaper/faster to send via state channels. Coinbase and Circle, two prominent firms authorized to trade USD/ETH, have launched an Ethereum-based coin pegged to USD called USDC to achieve this purpose (https://www.centre.io/pdfs/centre-whitepaper.pdf). The MakerDAO group has also released a token called DAI on Ethereum which achieves the USD peg via a decentralized system of smart contracts and collateralization (https://makerdao.com/whitepaper/).  
```

Comment:
- Stablecoin is probably one word in this case. 
- Might be worth mentioning Gemini and GUSD

Reply:
You're right about the stablecoin thing. I'll change it. I'll also mention GUSD.